RedditSharp/Things/Post.cs

```diff
@@ -225,14 +225,25 @@ public Task<Subreddit> GetSubredditAsync()
         /// </summary>
         /// <param name="limit">Maximum number of comments to return</param>
         /// <returns></returns>
-        public async Task<List<Comment>> GetCommentsAsync(int limit = 0)
+        public async Task<List<Comment>> GetCommentsAsync(int limit = 0, CommentSort sort = CommentSort.Best)
         {
             var url = string.Format(GetCommentsUrl, Id);
+
+            //Only 'best' comment sorting isn't named the same
```

Comment:
Fuck consistency amiright? Though it is Reddit so I'm not even surprised honestly..